README.md

```diff
@@ -39,14 +39,12 @@ compare Trusted Web Activity projects against the quality criteria.
 ## Community
 
 We welcome anyone who wants to contribute with issues, feedback, feature requests or just
-generally discuss Bubblewrap. There are a couple of other ways developers can participate
+generally discuss Bubblewrap. Those are a alternative ways developers can participate
```

Comment:
"Those are a alternative way" doesn't sound good (It should probably be "This is an alternative way").

However, I think it would read better as "Alternatively, developers can ..."

Reply:
updated